.github/actions/restore-npm-cache/action.yml

```diff
@@ -9,3 +9,6 @@ runs:
         path: "**/node_modules"
         key: cache-node-modules-${{ hashFiles('**/package-lock.json') }}
         restore-keys: cache-node-modules-npm run
+    - name: Prepare project
```

Comment:
Maybe move to a different workflow file or rename this one because this is not related to 'npm cache'

Reply:
This command is executed when we do an npm install, it is a pre-command, but since we are restoring the cache, it is not executed